lighthouse/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Dockerfile from https://github.com/GoogleChrome/lighthouse-ci/tree/main/docs/recipes/docker-client
+FROM node:12-buster-slim
```

Comment:
Please publish this Dockerfile into the metadrop docker hub namespace, so this image can be easily usable in another projects that does not use the boilerplate.

Reply:
@mistermoper I do not have permission to create a new repository on docker-hub, here is the repository https://github.com/Metadrop/docker-lighthouse-ci